MCP_Server/server.py

```
@@ -652,6 +678,158 @@ def load_drum_kit(ctx: Context, track_index: int, rack_uri: str, kit_path: str)
+        # Convert to grid
+        grid = grid_notation.notes_to_grid(notes)
+
+        return f"Clip: {clip_name} ({len(notes)} notes, {length} beats)\n\n{grid}"
+    except Exception as e:
+        logger.error(f"Error converting clip to grid: {str(e)}")
+        return f"Error converting clip to grid: {str(e)}"
+
+
+@mcp.tool()
+def grid_to_clip(
+    ctx: Context,
```

Ruff (0.14.10)

Unused function argument: `ctx`

(ARG001)